.gitignore

```diff
@@ -1,135 +1,139 @@
-.env
-.venv
-env/
-venv/
-ENV/
-env.bak/
-venv.bak/
-
-# Spyder project settings
-.spyderproject
-.spyproject
-
-# Rope project settings
-.ropeproject
-
-# mkdocs documentation
-/site
-
-# mypy
-.mypy_cache/
-.idea
-workspace/output/
-output
-test/optimize/test_fit
-test/test_files/text/
-test/
-test_autofit/optimize/test_fit/
-test_autofit/test_files/text/psycopg2-binary==2.8.1
-test_autofit/test_files/text/
-fit/test_autofit/optimize/test_fit
-*.DS_Store
-
-test_autofit/unit/config/priors/old
-
-# docs
-docs/_build
-docs/_static
-docs/_templates
-docs/generated
-docs/api/generated
-test_autofit/unit/samples.csv
-test_autofit/unit/non_linear/files/pickles/test.pickle
-test_autofit/unit/non_linear/nest/files/dynesty/output/dynesty_static[nlive_150__bound_multi_vol_dec_0.5_vol_check_2.0__enlarge_1.0__sample_auto]/samples/dynesty.pickle
+test_autofit/unit/tools/files/path/
+test_autofit/samples.csv
+.envr
+test_autofit/test_files/
+test_autofit/samples.csv
+__MACOSX
+*.swp
+test/autofit/test_fit
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+# Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+# Unit / coverage reports
+htmlcov/
+.tox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+local_settings.py
+db.sqlite3
+
+# Flask stuff:
+instance/
+.webassets-cache
+
+# Scrapy stuff:
+.scrapy
+
+# Sphinx documentation
+docs/_build/
+
+# PyBuilder
+target/
+
+# Jupyter Notebook
+.ipynb_checkpoints
+
+# pyenv
+.python-version
+
+# celery beat schedule file
+celerybeat-schedule
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
+.ropeproject
+
+# VS code
+.vscode
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+.idea
+workspace/output/
+output
+test/optimize/test_fit
+test/test_files/text/
+test/
+test_autofit/optimize/test_fit/
+test_autofit/test_files/text/psycopg2-binary==2.8.1
+test_autofit/test_files/text/
+fit/test_autofit/optimize/test_fit
+*.DS_Store
+
+test_autofit/unit/config/priors/old
+
+# docs
+docs/_build
+docs/_static
+docs/_templates
+docs/generated
+docs/api/generated
+test_autofit/unit/samples.csv
+test_autofit/unit/non_linear/files/pickles/test.pickle
+test_autofit/unit/non_linear/nest/files/dynesty/output/dynesty_static[nlive_150__bound_multi_vol_dec_0.5_vol_check_2.0__enlarge_1.0__sample_auto]/samples/dynesty.pickle
+>>>>>>> develop
```

Comment:
There's a stray conflict annotation here

Reply:
Yeah that shouldn't be there. What's the process for removing it from the pull request?